### ISSUE TITLE
Details List: Narrator doesn't announce level of list item

### ISSUE THREAD

**Issue by bcoard** [OfficeDev/office-ui-fabric-react#11820]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:

When a Narrator user navigates to an item in a Grouped Details List, the items level isn't announced.

### Please provide a reproduction of the issue in a codepen:

1. Navigate to [this](https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/grouped) page.
2. Turn Narrator on.
3. Using keyboard navigation go to an item in the list.
